diff -rup prado/framework/Data/ActiveRecord/TActiveRecordGateway.php prado-new/framework/Data/ActiveRecord/TActiveRecordGateway.php
--- framework/Data/ActiveRecord/TActiveRecordGateway.php	2021-08-24 14:37:07.000000000 +0200
+++ framework/Data/ActiveRecord/TActiveRecordGateway.php	2021-12-04 14:23:25.652942072 +0100
@@ -123,6 +123,11 @@ class TActiveRecordGateway extends \Prad
 	public function getTableInfo(TDbConnection $connection, $tableName)
 	{
 		$connStr = $connection->getConnectionString();
+		$connection->setActive(true);
+		$driver = $connection->getDriverName();
+		if($driver == 'pgsql') {
+			$tableName = strtolower($tableName);
+		}
 		$key = $connStr . $tableName;
 		if (!isset($this->_tables[$key])) {
 			//call this first to ensure that unserializing the cache
